cluster-scope/base/namespaces/m4d-blueprints/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
```

Comment:
Can we get the ticket label like it was done [here](https://github.com/operate-first/apps/blob/master/cluster-scope/base/namespaces/open-aiops/namespace.yaml#L8) Same for the other namespace.

Reply:
Done